### ISSUE TITLE
Running detect() or detect_multiple() on a webradio will make the script hang

### ISSUE THREAD

**Issue by outsbart** [CERN-CERT/WAD#35]
funny little bug

how to reproduce:
wad.Detector().detect("https://webradio.divsi.unimi.it")

**Comment by Feandil** [comment]
Funny little bug indeed.
Taking a quick look, the issue is, I think, that this URL returns an infinite chunked 'audio/mpeg'.
This means that the read at https://github.com/CERN-CERT/WAD/blob/master/wad/detection.py#L97 never returns.
I guess we should set up a sort of limit there...

**Comment by outsbart** [comment]
Exactly, one of the solution could be using [requests' iter_content](https://2.python-requests.org/en/master/api/#requests.Response.iter_content) and allowing the code to stop reading after a certain size threshold is reached.

Something like [this](https://stackoverflow.com/a/22347526)
